bolt/lib/Rewrite/RewriteInstance.cpp

```diff
@@ -1030,6 +1031,25 @@ void RewriteInstance::discoverFileObjects() {
       continue;
     }
 
+    if (BC->IsLinuxKernel && SymName == "linux_banner") {
```

Comment:
The reading part also should be in the `LinuxKernelRewriter`. You can access symbols via `BinaryData` objects.

Reply:
Done.